live-examples/js-examples/array-reverse.html

```diff
@@ -0,0 +1,10 @@
+<pre>
+<code id="static-js">var array1 = ['one', 'two', 'three'];
+var reversed = array1.reverse(); 
+
+console.log(array1);
+// expected output: Array ['three', 'two', 'one']
+console.log(reversed);
+// expected output: Array ['three', 'two', 'one']
+</code>
```

Comment:
I really like that this example shows the return value as well as the in-place array :).